db/src/Pos/DB/Rocks/Functions.hs

```diff
@@ -91,6 +93,11 @@ openNodeDBs recreate fp = do
         , lrcPath
         , miscPath
         ]
+
+    when (System.Info.os == "darwin") $
+        -- Prevent indexing of blocks on OSX
+        ensureEmptyFileExists (fp </> ".metadata_never_index")
```

Comment:
This is already on `develop`!
https://github.com/input-output-hk/cardano-sl/blob/develop/db/src/Pos/DB/Rocks/Functions.hs#L97-L99